qcodes/instrument_drivers/QuTech/IVVI.py

```diff
@@ -94,13 +131,24 @@ def __init__(self, name, address, reset=False, numdacs=16, dac_step=10,
             print('IVVI: get_all() failed, maybe connected to wrong port?')
             print(traceback.format_exc())
 
+        v = self.visa_handle
+
+        # make sure we igonore termination characters
+        # http://www.ni.com/tutorial/4256/en/#toc2 on Termination Character
+        # Enabled
+        v.set_visa_attribute(visa.constants.VI_ATTR_TERMCHAR_EN, 0)
+        v.set_visa_attribute(visa.constants.VI_ATTR_ASRL_END_IN, 0)
+
```

Comment:
And move the code which does the `get_all` to after the setting of the termination characters. First do the settings and then the communication.